src/stories/5-custom/filter/filter-checkbox-item.stories.tsx

```diff
@@ -26,6 +26,15 @@ const meta: Meta = {
 	},
 	argTypes: {
 		...CommonCustomStoryProps("filter-checkbox"),
+		label: {
+			description:
+				"A name/description of the purpose of the element. Accepts a string for a default label, or an object with a main label and a hint displayed in a popover.",
+			table: {
+				type: {
+					summary: "string | { mainLabel: string, hint: { content: string, zIndex?: number } }",
```

Comment:
also update for `filter-item`

```suggestion
					summary: "string | { mainLabel: string, hint?: { content: string, zIndex?: number } }",
```